src/main/java/core/basesyntax/WorkWithBytes.java

```diff
@@ -1,7 +1,27 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithBytes {
     public void writeBytesToFile(String fileName, byte[] data) {
-
+        File fileToWrite = new File(fileName);
+        try {
+            fileToWrite.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("File "
+                    + fileToWrite.getName() + " couldn't be created", e);
+        }
```

Comment:
doesn't it work without creating a new file?

Reply:
It does works